xtail.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: xtail
+  # This is really 2.1-9 upstream, so if we manually update this, we should fix
+  # the fetch URL below.
+  version: 2.1.9
+  epoch: 0
+  description: Tool to control multiple terminals from a single terminal
+  copyright:
+    - license: BSD-3-Clause
+    - license: Unlicense
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - autoconf
+      - automake
+      - build-base
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://salsa.debian.org/debian/xtail/-/archive/debian/2.1-9/xtail-debian-2.1-9.tar.bz2
+      expected-sha256: d6381635dd547f3f0bd1d0207f5095eef71b97f15039cfac43ef2288b488306e
+
+  - runs: mkdir -p ${{targets.destdir}}/usr/bin
+
+  - uses: autoconf/configure
+    with:
+      # Most autoconf/configure steps don't need the destdir, but this one does.
+      opts: |
+        --sysconfdir=${{targets.destdir}}/etc \
+        --prefix=${{targets.destdir}}/usr
```

Comment:
I'd love a second set of eyes on this in particular as I'm a noob wrt autoconf stuff.

The upstream repo is at: https://salsa.debian.org/debian/xtail